bitnami/cert-manager/values.yaml

```diff
@@ -201,6 +201,18 @@ controller:
   ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
   ##
   podAnnotations: {}
+  ## @param controller.dnsPolicy Pod DNS policy
+  ## Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  ##
+  dnsPolicy: ClusterFirst
+  ## @param controller.dnsConfig Pod DNS config
```

Comment:
This should be the same as the description in the README table. We have a tool to generate the README table from this metadata, you can read more about it here: https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#documentation-requirements

Reply:
Updated and recreated, using the required tool.